components/FinancialSummary/BarChartComponent.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState, useEffect, useRef } from 'react';
 import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
-import ExpensesLink from '../../config/finance/json-data/2024/ExpensesLink.json';
-import Expenses from '../../config/finance/json-data/2024/Expenses.json';
+const currentYear = String(new Date().getFullYear());
+const ExpensesLink = require(`../../config/finance/json-data/${currentYear}/ExpensesLink.json`);
+const Expenses = require(`../../config/finance/json-data/${currentYear}/Expenses.json`);
```

Comment:
Why are we moving the imports from module type to node js `require` imports?

Reply:
## If I use the normal imports, the compiler is throwing errors 

![image](https://github.com/asyncapi/website/assets/63534268/d837ee19-1433-4cbd-b503-ab45ac9669b9).

## However, We can use dynamic imports, but that will make the code more complicated because dynamic import returns a promise 
![image](https://github.com/asyncapi/website/assets/63534268/807634c4-ead9-47fa-847f-02a8be527b6d)

![image](https://github.com/asyncapi/website/assets/63534268/da195550-ede0-4bab-b525-acb23599de28)

## I would rather have to wrap the components into a then clause or into an async function

Reply:
Since, we are already moving into typescript soon, we should make these imports as dynamic. WDYT? @derberg

Reply:
yup, makes sense, @JeelRajodiya please adjust